lib/ansiblelint/utils.py

```diff
@@ -732,3 +739,108 @@ def get_rule_skips_from_line(line):
         noqa_text = line.split('# noqa')[1]
         rule_id_list = noqa_text.split()
     return rule_id_list
+
+
+def is_playbook(filename):
+    """
+    Given a filename, it should return true if it looks like a playbook. The
+    function is not supposed to raise exceptions.
+    """
+    # we assume is a playbook if we loaded a sequence of dictionaries where
+    # at least one of these keys is present:
+    playbooks_keys = {
+        "gather_facts",
+        "hosts",
+        "import_playbook",
+        "post_tasks",
+        "pre_tasks",
+        "roles"
+        "tasks",
+    }
+
+    # makes it work with Path objects by converting them to strings
+    if not isinstance(filename, six.string_types):
+        filename = str(filename)
+
+    try:
+        f = parse_yaml_from_file(filename)
+        if isinstance(f, AnsibleSequence) and playbooks_keys.intersection(f[0].keys()):
+            return True
+    except Exception as e:
+        print(
+            "Warning: Failed to load %s with %s, assuming is not a playbook."
+            % (filename, e))
+    return False
+
+
+def normpath(path):
+    """
+    Normalize a path in order to provide a more consistent output. Currently
```

Comment:
Please separate a 72-char title sentence from the rest of the docstring by an empty line as required by PEP257

Reply:
Including normalization in this review what not the best idea, I decided to create another PR at https://github.com/ansible/ansible-lint/pull/620 for implementing that, it would make review much easier. I will implement the changes on the new PR and rebase this once normalization lands.

Reply:
Re PEP257, lets address it the right way, and have CI check its compliance. See https://github.com/ansible/ansible-lint/pull/621 as a start.